tests/parallel.py

```diff
@@ -271,6 +271,18 @@ def test_jit_parallel_reducers(self):
         self.assertAlmostEqual(jn.square(np.mean(y.split(8), 0) - zmean).sum(), 0, places=8)
         self.assertAlmostEqual(jn.square(np.sum(y.split(8), 0) - zsum).sum(), 0, places=8)
 
+    def test_trainvar_assign(self):
+        m = objax.ModuleList([objax.TrainVar(jn.zeros(2))])
+
+        def increase():
+            m[0].assign(m[0].value + 1)
+            return m[0].value
+
+        para_increase = objax.Parallel(increase, m.vars())
+        with m.vars().replicate():
+            para_increase()
+        self.assertEqual(m[0].value.tolist(), [1., 1.])
```

Comment:
Maybe add another test which does increase by different values in different replicas:
```

def increase(x):
  m[0].assign(m[0].value + x)
  return m[0].value

x = np.arange(8)
para_increase = objax.Parallel(increase, m.vars())
with m.vars().replicate():
  para_increase()

...

```

Reply:
Done